website/docs/components/form/masked-input/partials/code/component-api.md

```diff
@@ -10,6 +10,9 @@ The Masked Input component has two different variants with their own APIs:
 <Doc::ComponentApi as |C|>
   <C.Property @name="value" @type="string|number|date">
     Input control’s value.
+    <br/>
+    <br/>
+    _Notice: if the string provided contains newline characters they will be stripped, unless `@isMultiline` is set to `true` (this renders the input as a `<textarea>` element, which supports multi-line text)._
```

Comment:
This is good but the only thing I wonder is if the entire test should use italics? Do we have other examples like that?

Reply:
Yes, I tend to use it to differentiate (and make it "pop" a bit) a "notice" message :) 
<img width="523" alt="screenshot_3380" src="https://github.com/hashicorp/design-system/assets/686239/844d76e4-cac5-4617-913f-bd800091a8e2">